boxwise_flask/auth_helper.py

```diff
@@ -118,3 +126,32 @@ def decorated(*args, **kwargs):
         )
 
     return decorated
+
+
+def get_id_token():
+    # TODO: pass entire id_token in under x-boxwise header, and decode here
+    email = request.headers.get("X-Boxwise", None)
```

Comment:
hey @mcgnly :wave: a quick question so I understand the context of the PR. How is the `X-Boxwise` header intended to be used vs the `Authorization` header that `get_token_auth_header` is using?